maui-toolkit/Otp-Input/Styling-Modes.md

```diff
@@ -58,7 +58,7 @@ SfOtpInput otpInput = new SfOtpInput()
 
 ## Underline mode
 
-You can customize the appearance of input fields with an underline by setting the `StylingMode` property to `Underlined.`
+You can customize the appearance of input fields with an underline by setting the [StylingMode](https://help.syncfusion.com/cr/maui-toolkit/Syncfusion.Maui.Toolkit.OtpInput.OtpInputStyle.html) property to [Underlined.](https://help.syncfusion.com/cr/maui-toolkit/Syncfusion.Maui.Toolkit.OtpInput.OtpInputStyle.html#Syncfusion_Maui_Toolkit_OtpInput_OtpInputStyle_Underlined)
```

Comment:
Add correct link for styling mode - https://help.syncfusion.com/cr/maui-toolkit/Syncfusion.Maui.Toolkit.OtpInput.SfOtpInput.html#Syncfusion_Maui_Toolkit_OtpInput_SfOtpInput_StylingMode